metagraph/src/annotation/annotation_converters.cpp

```diff
@@ -1172,6 +1173,20 @@ void convert_to_row_diff(const std::vector<std::string> &files,
             return;
         }
         mem_bytes -= anchor_size;
+
+        const std::string rd_succ_fname = graph_fname + kRowDiffForkSuccExt;
+        if (!fs::exists(rd_succ_fname)) {
+            logger->error("Can't find row-diff successor bitmap at {}", rd_succ_fname);
+            exit(1);
+        }
+        uint64_t rd_succ_size = fs::file_size(rd_succ_fname);
+        if (rd_succ_size > mem_bytes) {
+            logger->warn("row-diff successor bitmap ({} MiB) is larger than"
```

Comment:
Why is this just a warning instead of an error?

Reply:
I think the idea is to let it run if the process doesn't get killed.